src/index.ts

```diff
@@ -52,8 +62,9 @@ const getSoupLanguageData = async (soupRepoUrl: string) => {
   if (soupLanguagesGitHubResponse.status !== 200) return 'unknown'
   const soupLanguagesData = soupLanguagesGitHubResponse.data
 
-  const totalSoupBytes = Object.values(soupLanguagesData).reduce(
-    (a: number, b: number) => a + b
+  const totalSoupBytes = Object.values(soupLanguagesData)?.reduce(
+    (a: number, b: number) => a + b,
+    0
   )
```

Comment:
`totalSoupBytes` can still be undefined, add `?? 0`

```suggestion
  const totalSoupBytes = Object.values(soupLanguagesData)?.reduce(
    (a: number, b: number) => a + b,
    0
  ) ?? 0
```